Apps/WebClient/src/ClientApp/src/views/validateEmail.vue

```diff
@@ -51,46 +61,22 @@ export default class ValidateEmailView extends Vue {
 <template>
     <b-container>
         <b-row class="pt-5">
-            <b-col class="text-center mb-5">
-                <h4 v-if="isLoading" class="title">
-                    We are verifying your email...
-                </h4>
-                <h4
-                    v-if="!isLoading && isSuccess === true"
-                    class="text-success"
-                >
+            <b-col class="text-center mb-5 title">
+                <h4 v-if="isLoading">We are verifying your email...</h4>
+                <h4 v-else-if="isSuccess" class="text-success">
                     Your email was successfully verified!
                 </h4>
-                <h4
-                    v-if="!isLoading && isSuccess === false"
-                    class="text-danger"
-                >
+                <h4 v-else-if="isActionRequired">
```

Comment:
How do we know "isActionRequired == true" will lead to expired email, it may be user has not validated invite link yet but the link is still valid (or non-expired).

Reply:
The service returns 3 states, success == "validated", error == "something wrong with the invite key", actionrequired == "expired"